superbench/benchmarks/model_benchmarks/model_base.py

```diff
@@ -374,7 +374,7 @@ def _is_finished(self, curr_step, curr_time):
 
         if (
             (self._args.duration > 0 and (curr_time - self._sub_benchmark_start_time) >= self._args.duration)
-            or (total_steps > 0 and curr_step >= total_steps)
+            or (self._args.num_steps > 0 and curr_step >= total_steps)
```

Comment:
the previous design is when 'num_warmup + num_steps < 0' and duration>0 it only use the duration, now you want to change to when 'num_step < ' and duration >0, it will only use the duration for convenience, if so, you need to change the doc accordingly

Reply:
Yes, it is more clear to only use num_step in case users will use a big value for num_warmup to affect the value of num_warmup + num_steps.
Thanks, I will revise the doc.

Reply:
Done.